src/actionCreators/index.js

```diff
@@ -19,31 +19,71 @@
 //
 
 import axios from 'axios';
-import { uploadStarted, uploadCompleted } from '../actions';
+import {
+  jobCreating,
+  jobCreated,
+  jobCreationFailed,
+  jobProcessing,
+  jobCompleted,
+} from '../actions';
+import { API } from '../constants';
 
-export const createSigningJob = files => dispatch => {
-  console.log('** HERE **', files);
+axios.defaults.baseURL = API.BASE_URL();
```

Comment:
mutating global variable like doens't seem to be right to me. maybe you can create an instance of an axios?
const instance = axios.create({
  baseURL: API_BASE_URL,
});

Reply:
Fixed.